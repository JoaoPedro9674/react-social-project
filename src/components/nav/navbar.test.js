import React from 'react';
import renderer from 'react-test-renderer';

import Navigation from './navbar';

describe('<Navigation/>', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Navigation />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should show a login option with user state', () => {
    const component = renderer.create(<Navigation />);
    const tree = component.toJSON();

    expect(tree.type).toBe('nav');
    expect(tree).toMatchSnapshot();
  });
});
