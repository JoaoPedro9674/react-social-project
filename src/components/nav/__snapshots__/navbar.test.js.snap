exports[`<Navigation/> should render correctly 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top">
  <div
    className="container">
    <div
      className="row middle-xs center-xs around-xs">
      <div
        className="col-sm-4 col-sm-offset-4 col-xs-8">
        <div
          className="logo"
          onClick={[Function]}
          style={
            Object {
              "fontSize": "2.3em",
            }
          }>
          <span
            style={
              Object {
                "color": "#4F93E0",
              }
            }>
            L
          </span>
          <span
            style={
              Object {
                "color": "#D0021B",
              }
            }>
            e
          </span>
          <span
            style={
              Object {
                "color": "#F8E71C",
              }
            }>
            t
          </span>
          <span
            style={
              Object {
                "color": "#4F93E0",
              }
            }>
            t
          </span>
          <span
            style={
              Object {
                "color": "#417505",
              }
            }>
            e
          </span>
          <span
            style={
              Object {
                "color": "#D0021B",
              }
            }>
            r
          </span>
          <span
            style={
              Object {
                "color": "#F8E71C",
              }
            }>
            s
          </span>
          <small
            className="social">
            .social
          </small>
        </div>
      </div>
      <div
        className="col-sm-4 col-xs-4">
        <div>
          <button
            className="btn btn-default"
            onClick={[Function]}
            type="button">
            Log in or sign up
          </button>
        </div>
      </div>
    </div>
  </div>
</nav>
`;

exports[`<Navigation/> should show a login option with user state 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top">
  <div
    className="container">
    <div
      className="row middle-xs center-xs around-xs">
      <div
        className="col-sm-4 col-sm-offset-4 col-xs-8">
        <div
          className="logo"
          onClick={[Function]}
          style={
            Object {
              "fontSize": "2.3em",
            }
          }>
          <span
            style={
              Object {
                "color": "#4F93E0",
              }
            }>
            L
          </span>
          <span
            style={
              Object {
                "color": "#D0021B",
              }
            }>
            e
          </span>
          <span
            style={
              Object {
                "color": "#F8E71C",
              }
            }>
            t
          </span>
          <span
            style={
              Object {
                "color": "#4F93E0",
              }
            }>
            t
          </span>
          <span
            style={
              Object {
                "color": "#417505",
              }
            }>
            e
          </span>
          <span
            style={
              Object {
                "color": "#D0021B",
              }
            }>
            r
          </span>
          <span
            style={
              Object {
                "color": "#F8E71C",
              }
            }>
            s
          </span>
          <small
            className="social">
            .social
          </small>
        </div>
      </div>
      <div
        className="col-sm-4 col-xs-4">
        <div>
          <button
            className="btn btn-default"
            onClick={[Function]}
            type="button">
            Log in or sign up
          </button>
        </div>
      </div>
    </div>
  </div>
</nav>
`;
